CRP-1752: Document notebook dataset scheduled reports (#32908)
DataDog/documentation: content/en/notebooks/advanced_analysis/_index.md

3. Select the number of rows you want to export (up to the maximum available).
4. The CSV file automatically downloads to your computer.

### Schedule a CSV report
With scheduled reports, you can automatically receive query results for analysis cells through email or Slack.

To schedule a report on an analysis cell, select **Schedule report** from the dropdown next to **Save to Dashboard**.

{{< img src="/notebooks/analysis_features/schedule_report_from_cell.png" alt="Example of scheduling a report from an analysis cell in a notebook" style="width:100%;" >}}

In the configuration modal that opens, set a schedule to determine when and how often the report is sent.

> {{< img src="/notebooks/analysis_features/select_schedule.png" alt="Schedule configuration section of report scheduling modal" style="width:100%;" >}}

Reviewer: ```suggestion
To schedule a report on an analysis cell:

1. Select **Schedule report** from the dropdown next to **Save to Dashboard**.

   {{< img src="/notebooks/analysis_features/schedule_report_from_cell.png" alt="A report view showing a dropdown menu with the 'Schedule report' option highlighted" style="width:100%;" >}}

1. In the configuration modal that opens, set a schedule to determine when and how often the report is sent.
```

{{< img src="/notebooks/analysis_features/add_email_recipients.png" alt="Email recipient section of report scheduling modal" style="width:100%;" >}}

Here's an example report email, including the optional 10-row inline preview:
{{< img src="/notebooks/analysis_features/report_email.png" alt="Example report email" style="width:100%;" >}}

**Slack:** To add a Slack channel as recipient, select the Slack workspace and channel from the available dropdowns. If you do not see any available workspaces, ensure you have the Datadog [Slack Integration][6] installed. All public channels within the Slack workspace should be listed automatically. To select a private Slack channel, make sure to invite the Datadog Slack bot to the channel in Slack. To send a test message to Slack, add a channel recipient and click **Send Test Message.**

> {{< img src="/notebooks/analysis_features/add_slack_recipients.png" alt="Email recipient section of report scheduling modal" style="width:100%;" >}}

Reviewer: ```suggestion
   {{< img src="/notebooks/analysis_features/add_slack_recipients.png" alt="Email recipient section of report scheduling modal" style="width:100%;" >}}
```

> 

Reviewer: ```suggestion
1.  Save your schedule.

```

> You can view, search, edit, and delete existing report schedules from the **Reports** tab.

Reviewer: ```suggestion
You can view, search, edit, and delete existing report schedules from the **Reports** tab.

{{< img src="/notebooks/analysis_features/reports_page.png" alt="Filtered view of the Reports tab" style="width:100%;" >}}

```

To schedule reports and view other users' report schedules, users need the **CSV Report Schedules Write** permission. To edit other users' report schedules, users need the **CSV Report Schedules Manage** permission. These can be granted by a user with the **Org Management** permission.

> {{< img src="/notebooks/analysis_features/reports_page.png" alt="Filtered view of the Reports tab" style="width:100%;" >}}

Reviewer: Moving this up

```suggestion
```
